src/ts_catalog/continuous_agg.c

```diff
@@ -792,7 +792,7 @@ ts_continuous_agg_hypertable_status(int32 hypertable_id)
 {
 	ScanIterator iterator =
 		ts_scan_iterator_create(CONTINUOUS_AGG, AccessShareLock, CurrentMemoryContext);
-	ContinuousAggHypertableStatus status = HypertableIsNotContinuousAgg;
+	ContinuousAggHypertableStatus status = { false, false };
```

Comment:
Probably would be better to memset() it, it is actually not clear how compiler will init this structure. Alternative approach could be using union to init